.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,201 @@
+name: AWS Commercial Canaries
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "*"
```

Comment:
Is this correct to run this workflow for all branches?

Reply:
That's the plan. Riffraff is currently setup so that pushes to main will deploy the production canary and deploys to other branches will deploy the test canary. I felt like this would be a better solution that copying and pasting code to the test canary during testing (although you can still do this). The canaries will be worked on infrequently so we are unlucky to have multiple people working on multiple canary tickets and be stepping on each others toes, so this strategy should be OK.

https://riffraff.gutools.co.uk/deployment/continuous
![image](https://user-images.githubusercontent.com/9574885/168783548-13b33a53-2a9b-432a-97a1-134ce3cd7cfb.png)

Reply:
Nice, thanks for the detailed explanation!